src/ol_infrastructure/infrastructure/elasticsearch/Pulumi.infrastructure.elasticsearch.mitx_online.Production.yaml

```diff
@@ -0,0 +1,10 @@
+secretsprovider: awskms://alias/PulumiSecrets
+encryptedkey: AQICAHionUR8LBW1ALuVC0rCH3AE2oQIfGMCx3XmpDH9HjM2LQEgP2bg0CBiwICatIkzeQCYAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMI6TROqUcQjK1/ia0AgEQgDu1fsY0A3yaABRuCONUf7x6d6a7+mJ+hiLUQx9dZB105RzS9k7LNsQuHNgROREE91RGlQfDCr2hbLY0dA==
+config:
+  aws:region: us-east-1
+  elasticsearch:instance_type: medium
```

Comment:
We'll probably want a larger instance type for production, especially since we'll actually be using it between the forums and course discovery.